### ISSUE TITLE
Text cuttoff in cocoon

### ISSUE THREAD

**Issue by yougotashovel** [photonstorm/phaser#1561]
Text appears to be cutting off in Cocoonjs canvas+, working fine in browsers.  Using the default font it gets cut off at the bottom (slightly), and using my own custom font it gets cut off at the top.

Tried the following to no result:
* Changing lineSpacing just adds space inbetween text
* Changing text size doesn't make a difference

Anyone with any ideas what could be causing it?

Using version 2.2.2

**Comment by yougotashovel** [comment]
Okay, i've isolated this issue to Phaser, it's not with CocoonJS.

I tested using Canvas+ without Phaser, using my custom font, and the regular getContext method of adding text, and there's no issues.  Text appears as it should, nothing cut-off.

1. I get the same cutoff problem with Phaser using either Webview or Canvas+.
2. The issue disappears for Webview if i change the Text size to 'em' instead of 'px', but it gets worse in Canvas+, it defaults to another font and the text is tiny.
3. I rebuilt the font in another app, and the problem persists.

I think the problem is something to do with the height calculation in phaser, possibly if the font is narrow.  Letters get cut-off somewhere above the x-height set in the font, regardless of font size.

Has anybody experienced anything similar? Or know of a possible solution?  Thanks
